### ISSUE TITLE
E2E tests warning appears without e2e in the project.

### ISSUE THREAD

**Issue by amiryonatan** [wix/yoshi#764]
# 🐛 Bug Report

### 😯 Current Behavior

When running npm test in our project, we started receiving the following warning:
Warning:

  you are running e2e tests and doesn’t have any bundle located in the statics directory
  you probably need to run npx yoshi build before running the tests

But there are no e2e tests in the project.

### 🤔 Expected Behavior

Warning should not be visible when no e2e tests exists.

### 👣 Steps to Reproduce

clone adi-client monorepo:
```
git clone https://github.com/wix-private/adi-client
cd adi-client/smart-assist
npm i && npm run build && npmt
```

**Comment by ranyitz** [comment]
Thanks @amiryonatan!

I think this issue happens only when you use `jest`.

I'll try to explain the issue so it will be easier to apply a fix:
https://github.com/wix/yoshi/blob/a05077e7e389e040750aff32191bc4000b2c0ed0/packages/yoshi/src/commands/test.js#L177-L180

The above condition shows that when using `jest` without `watch` yoshi will always bootstrap the cdn, which in turn performs a check for a bundle in statics directory.

https://github.com/wix/yoshi/blob/a05077e7e389e040750aff32191bc4000b2c0ed0/packages/yoshi/src/commands/test.js#L61-L73

The problem is that we bootstrap cdn for a case where it's not needed.

We can add a check for `e2e` tests, but as the glob pattern for e2e tests is different from the protractor one:

https://github.com/wix/yoshi/blob/a05077e7e389e040750aff32191bc4000b2c0ed0/packages/jest-yoshi-preset/jest-preset.js#L31

We'll need to extract it to `yoshi-config` and use it in both places.

PRs welcome!

**Comment by ranyitz** [comment]
Fixed on #775
